.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/opensearch-sdk
+*   @opensearch-project/opensearch-sdk-java
```

Comment:
I did change the name of the team from `opensearch-sdk` to `opensearch-sdk-java`